app/controllers/application_controller.rb

```diff
@@ -7,6 +7,7 @@
 require 'sentry_logging'
 require 'aes_256_cbc_encryptor'
 
+# random change to app/
```

Comment:
Will these random changes will be removed before merging the PR?

Reply:
Yep